deltametrics/mask.py

```diff
@@ -139,3 +572,320 @@ def a_land_function(self):
         """This is a wrapper to the below function.
         """
         return a_land_function(self.data)
+
+
+class ShoreMask(BaseMask, OAM):
```

Comment:
Would this be better named `ShorelineMask`? I don't feel strongly about this, just a thought